api/server_test.go

```diff
@@ -100,6 +100,10 @@ func TestServerRoutes(t *testing.T) {
 		{"GET", "/list/madeupname", nil, 500},
 		{"GET", "/list/madeupname/datasetname", nil, 400},
 		// {"GET", "/list/[peername]", {}, {proper response}, 200}
+		{"OPTIONS", "/history/", nil, 200},
+		{"GET", "/history/", nil, 400},
+		// {"GET", "/history/me/[datasetname]", {}, {proper response},  200},
```

Comment:
I was saying to @osterbit that in future PR's we should get in the habit of adding comments on commented-out tests that explain _why_ a test is currently disabled. Weather it's unfinished, because of a bug, etc. So no worries for this PR, but in the future I'm going to start requesting we do this.